htmlhint-server/src/server.ts

```diff
@@ -79,10 +80,20 @@ function makeDiagnostic(problem: htmlhint.Error, lines: string[]): server.Diagno
  */
 function getConfiguration(filePath: string): any {
     var options: any;
-    if (settings.htmlhint && settings.htmlhint.options && Object.keys(settings.htmlhint.options).length > 0) {
-        options = settings.htmlhint.options;
-    }
-    else {
+    if (settings.htmlhint) {
+        if (settings.htmlhint.configFile && settings.htmlhint.options && Object.keys(settings.htmlhint.options).length > 0) {
+            throw new Error(`The configuration settings for HTMLHint are invalid. Please specify either 'htmlhint.configFile' or 'htmlhint.options', but not both.`);
+        }
+        if (settings.htmlhint.configFile) {
+            if (fs.existsSync(settings.htmlhint.configFile)) {
+                options = loadConfigurationFile(settings.htmlhint.configFile)
+            } else {
+                throw new Error(`The configuration settings for HTMLHint are invalid. The file '${settings.htmlhint.configFile}' specified in 'htmlhint.configFile' could not be found.`);
```

Comment:
if settings.htmlhint.configFile is a relative path, then it will probably be helpful to output both the `configFile` value, and the wahtever the fully resolved path is - I think in node this is `path.resolve()`?

Reply:
or, just the fully resolved path.

Reply:
Yeh, nice suggestion, I've a hint when the path is relativ.